hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -389,43 +389,51 @@
                 its join request will be rejected if it is not a trusted member.
                 You can give an IP addresses range using the wildcard (*) on the last digit of
                 IP address (e.g. 192.168.1.* or 192.168.1.100-110).
-        - <tcp>:
-            Set its "enabled" attribute to true for discovery by TCP/IP. It has the following sub-elements
-            to fine tune the multicast discovery.
+        - <tcp-ip>:
+            It has the following attributes.
+            - "enabled":
+                Specifies whether the TCP/IP discovery is enabled or not. Default value is false.
+            - "connection-timeout-seconds":
+                The maximum amount of time Hazelcast is going to try to connect to a well known member
+                before giving up. Setting it to a too low value could mean that a member is not able
+                to connect to a cluster. Setting it to a too high value means that member startup could
+                slow down because of longer timeouts (e.g. when a well known member is not up). Increasing
+                this value is recommended if you have many IPs listed and the members cannot properly
+                build up the cluster. Its default value is 5.
+
+            It has the following sub-elements to fine tune the multicast discovery.
```

Comment:
omit `multicast` here (and in the same passage below)

Reply:
Done.